guides/0000-atomic-swap.md

```diff
@@ -0,0 +1,97 @@
+### What is atomic swap
+
+Atomic swap is a feature where two different users can exchange different tokens in the same transaction without any third party and any trust.
+
+For instance, imagine you want to buy an NFT and pay with HTR. This can be done in a single transaction with no trust between the buyer and the seller.
+
+### Operations flow
+
+In the example described here we will assume that there is a central backend to coordinate the atomic swap. Let's assume we have two users (user1 and user2), each one with their own wallets (W1 and W2) in their devices and they want to swap token1 and token2, i.e. user1 will send token1 from W1 to W2 and user2 will send token2 from W2 to W1. See the image below.
```

Comment:
There are other possible configurations. For example, the wallets might exchange messages through a broker (e.g., STUN server). In this case, the wallets can coordinate and do an atomic swap.